modevo-consistency/src/main/java/giis/modevo/consistency/OracleCSV.java

```diff
@@ -0,0 +1,392 @@
+package giis.modevo.consistency;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.datastax.oss.driver.api.core.cql.BoundStatement;
+import com.datastax.oss.driver.api.core.cql.ColumnDefinitions;
+import com.datastax.oss.driver.api.core.cql.PreparedStatement;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import com.datastax.oss.driver.api.core.cql.Row;
+
+import giis.modevo.migration.script.ScriptException;
+import giis.modevo.migration.script.execution.CassandraConnection;
+import giis.modevo.model.DocumentException;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Class of the oracle that contains all methods related to the CSVs used to compare the data from the SQL database and the Cassandra
+ * database
+ */
+@Slf4j
+public class OracleCSV {
+	private static final String CSV_OPEN = "Error proccesing the CSV  file ";
+
+	/**
+	 * Method that returns the data of the database into a csv
+	 */
+	public String csvCassandra(String nameTable, String keyspace, String properties) {
+		CassandraConnection connection;
+		connection = new CassandraConnection(properties);
+		// ResultSet for cassandra, different than SQL ones
+		ResultSet rs = connection.executeStatement("SELECT * FROM \"" + keyspace + "\"." + nameTable + ";");
+		File file;
+		String nameFile = "dat/out/" + keyspace + "/" + nameTable + "CQL.csv";
```

Comment:
If I remember well, I have see this in another places. A method that makes some processing, wrtining the result to a file, should not construct the file name in his body, because this causes that the convention used for the files be scattered across the source code. The test classes should be the only ontes that know that `dat/out` is the folder where the results are stored.